packages/fields/src/types/Integer/views/Controller.js

```diff
@@ -1,21 +1,26 @@
 // @flow
 import FieldController from '../../../Controller';
 
-type FilterGraphQL = {| type: string, value: string |};
-type FilterLabel = {| label: string |};
-type FormatFilter = {| label: string, value: string |};
+type FilterGraphQL = {| path: string, type: string, value: string |};
+type FilterLabel = {| label: string, type: string |};
+type FormatFilter = {| label: string, type: string, value: string |};
 type DataType = { [key: string]: string };
 
-export default class TextController extends FieldController {
-  getFilterGraphQL = ({ type, value }: FilterGraphQL): string => {
-    const key = type === 'is' ? `${this.path}` : `${this.path}_${type}`;
-    return `${key}: ${value}`;
+export default class IntegerController extends FieldController {
+  getFilterGraphQL = ({ path, type, value }: FilterGraphQL): string => {
+    const key = type === 'is' ? path : `${path}_${type}`;
+    let arg = value.replace(/\s/g, '');
+    if (['in', 'not_in'].includes(type)) {
+      arg = `[${arg}]`;
+    }
+    return `${key}: ${arg}`;
   };
-  getFilterLabel = ({ label }: FilterLabel): string => {
-    return `${this.label} ${label.toLowerCase()}`;
+  getFilterLabel = ({ label, type }: FilterLabel): string => {
+    const suffix = ['in', 'not_in'].includes(type) ? ' (comma separated)' : '';
```

Comment:
I think it's probably easier if we just put this `suffix` explicitly on the labels at line 70 and 75, it'll keep this code nice and simple.

Reply:
I agree, that would be better. I was trying to keep the code consistent with the UUID and MongoID fields.

Reply:
Looks like adding '(comma separated)' directly on the field label results in a UI where the suffix is added to the filter drop-down and the active filter pill interface. It seems like the term `label` is overloaded for this context.

For this bug fix, I suggest we leave this implementation as it's consistent with other fields. I suggest refactoring fields to have an option to override the input placeholder value.

The FieldController.getFilterTypes could return a list of types with the included placeholder:
```
getFilterTypes = () => [
    {
      type: 'in',
      label: 'Is one of',
      placeholder: 'Is one of (comma separated)',
      getInitialValue: () => '',
    }
]
```